src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestTrailersLayoutRenderer.cs

```diff
@@ -0,0 +1,93 @@
+﻿#if ASP_NET_CORE3
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Request Trailers
+    /// </summary>
+    /// <example>
+    /// <para>Example usage of ${aspnet-request-trailers}</para>
+    /// <code lang="NLog Layout Renderer">
+    /// ${aspnet-request-trailers:OutputFormat=Flat}
+    /// ${aspnet-request-trailers:OutputFormat=JsonArray}
+    /// ${aspnet-request-trailers:OutputFormat=JsonDictionary}
+    /// ${aspnet-request-trailers:OutputFormat=JsonDictionary:TrailerNames=username}
+    /// ${aspnet-request-trailers:OutputFormat=JsonDictionary:Exclude=access_token}
+    /// </code>
+    /// </example>
+    [LayoutRenderer("aspnet-request-trailers")]
+    public class AspNetRequestTrailersLayoutRenderer : AspNetLayoutMultiValueRendererBase
+    {
+        /// <summary>
+        /// Trailer names to be rendered.
+        /// If <c>null</c> or empty array, all trailers will be rendered.
+        /// </summary>
+        public List<string> TrailerNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets the keys to exclude from the output. If omitted, none are excluded.
+        /// </summary>
+        /// <docgen category='Rendering Options' order='10' />
+        public ISet<string> Exclude { get; set; } = 
+            new HashSet<string>(new[] { "ALL_HTTP", "ALL_RAW", "AUTH_PASSWORD" }, StringComparer.OrdinalIgnoreCase);
```

Comment:
Are these standard Trailers-values ?

Reply:
I think that was accidental copy from request header, I will remove them